### DISCUSSION TITLE
patterns.css

### DISCUSSION THREAD

**Discussion by argyleink** [argyleink/open-props#66] [Ideas]
best if they're one-liners

**Comment by argyleink** [comment]
`repeating-conic-gradient(red 0% 25%, tan 0% 50%) 50%/ 4em 4em` - https://codepen.io/thebabydino/full/OJmpzya

**Reply by argyleink** [reply]
more from Ana! https://css-tricks.com/background-patterns-simplified-by-conic-gradients/

**Comment by argyleink** [comment]
https://bansal.io/pattern-css

**Comment by Afif13** [comment]
I made a small website with patterns that can be interesting: https://github.com/Afif13/CSS-Pattern

For the sake of simplicity, I am using `background` and `background-size` but we can integrate the size in the shorthand to have a one line code.
